File: year2020/month08/day0821/gebal_0821.c
#include<stdio.h>

int main() {
	int cache[2][2][4], num;
	for (int n = 0; n < 2; n++) {
		if (n == 0) printf("first array\n");
		else printf("second array\n");
		for (int i = 0; i < 2; i++) {
			for (int j = 0; j < 4; j++) {
				scanf(" %d", &num);
				cache[n][i][j] = num;
			}
		}
	}
	for (int i = 0; i < 2; i++) {
		for (int j = 0; j < 4; j++) {
			printf("%d ", cache[0][i][j] * cache[1][i][j]);
		}
		printf("\n");
	}
}

File: year2020/month08/day0821/gebal_0821.py
import numpy as np
first = []
second = []
print("first array")
[first.append(input().split(' ')) for x in range(2)]
print("second array")
[second.append(input().split(' ')) for i in range(2)]
first = np.array(first, np.int32)
second = np.array(second, np.int32)
[[print(v, end=' ') if i != 3 else print(v) for i, v in enumerate(x)] for x in first * second]
